client/src/components/AuthContainer/AuthContainer.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box, Grid, Paper } from '@mui/material';
+import useStyles from './useStyles';
+
+type Props = {
+  children: JSX.Element | JSX.Element[] | string | string[];
```

Comment:
We could just put JSX.Element? I don't see a case where we are passing a list of jsx elements or a string or a list of strings. 🤔

Reply:
We want to be more strict with our types and our expectation is only a JSX element being the child.